scipy/stats/tests/test_stats.py

```diff
@@ -4450,6 +4450,15 @@ def test_nist(self):
             assert_allclose(res[0], f, rtol=rtol,
                             err_msg='Failing testcase: %s' % test_case)
 
+    def test_constant_values(self):
+        # For more details, look on https://github.com/scipy/scipy/issues/11669
+        a = np.array([42, 42, 42], dtype=np.uint16)
+        b = np.array([7, 7, 7, 7], dtype=np.uint16)
```

Comment:
Just checking, is there a particular reason for specifying the `dtype` as `np.uint16`? 
I'm guessing the answer is no.

Reply:
No, there isn't. Are You suggesting that in such case it shouldn't be specified?